app/controllers/user.js

```diff
@@ -15,3 +17,17 @@ exports.register = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.signIn = (req, res, next) =>
+  userService
+    .findOne({ email: req.body.email })
+    .then(user => {
+      if (user) {
+        if (encryptionService.validatePasssword(req.body.password, user.password)) {
+          res.writeHead(200, { token: authenticationService.generateToken(user) });
```

Comment:
Use a header for this, headers are used like [this](https://nodejs.org/api/http.html#http_response_setheader_name_value), also the header token is non-standard way of return a token